src/charts/draw/chartobject_remove.js

```diff
@@ -99,8 +99,33 @@ define(["jquery", "datatables", 'charts/charts'], function ($) {
 
             $(".indicator_remove_dialog").data('refererChartID', containerIDWithHash).dialog("open");
 
-        }
+        },
+
+        showDialog : function(containerIDWithHash){
+            var chart = $(containerIDWithHash).highcharts(),
+                middle = chart.chartWidth/2 - 130,
+                text = "Double click the object to remove.";
+            
+            chart.display = chart.renderer.label(text, chart.plotLeft + middle, chart.plotTop)
+                .attr({
+                    fill: Highcharts.getOptions().colors[0],
+                    padding: 10,
+                    r: 5,
+                    zIndex: 8,
+                    opacity: 0.95
+                })
+                .css({
+                    color: '#FFFFFF'
+                })
+                .add();
 
+            setTimeout(function () {
```

Comment:
@arnabk I'm just displaying a message that fades away in 2 seconds. Since the message shouldn't be always displayed on the chart (to avoid this message from interfering with the view). I'am removing it in the next step.